### PR TITLE
add consoleLink api for header to consume

### PR DESCRIPTION
**Pull request by willkutler**
https://github.com/open-cluster-management/backlog/issues/11882

### CODE REVIEW

File: server/lib/server/console-links.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+const request = require('request').defaults({ rejectUnauthorized: false }),
+      config = require('../../config'),
+      fs = require('fs'),
+      log4js = require('log4js'),
+      logger = log4js.getLogger('service-account'),
+      serviceAccountPath = '/var/run/secrets/kubernetes.io/serviceaccount',
+      _ = require('lodash')
+
+module.exports.getConsoleLinks = (req, cb) => {
```
Comment (JustinKuli) [inline server/lib/server/console-links.js:10]: What's the `req` here? I never see it used, but I'm not sure if I'm just missing something.

**Review by JustinKuli — APPROVED** [review]
/lgtm

**Review by JustinKuli — APPROVED** [review]
/lgtm

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Pull request merged**